content/guides/caching.md

```diff
@@ -46,63 +52,67 @@ module.exports = {
 Running webpack with this config will produce the following output:
 
 ```bash
-Hash: 55e783391098c2496a8f
-Version: webpack 1.10.1
-Time: 58ms
-Asset Size Chunks Chunk Names
-main.55e783391098c2496a8f.js 1.43 kB 0 [emitted] main
-vendor.55e783391098c2496a8f.js 1.43 kB 1 [emitted] vendor
-[0] ./src/index.js 46 bytes {0} [built]
-[0] ./src/vendor.js 40 bytes {1} [built]
+Hash: 2a6c1fee4b5b0d2c9285
+Version: webpack 2.2.0
+Time: 62ms
+                         Asset     Size  Chunks             Chunk Names
+vendor.2a6c1fee4b5b0d2c9285.js  2.58 kB       0  [emitted]  vendor
+  main.2a6c1fee4b5b0d2c9285.js  2.57 kB       1  [emitted]  main
+   [0] ./src/index.js 63 bytes {1} [built]
+   [1] ./src/vendor.js 63 bytes {0} [built]
 ```
 
-But the problem here is that, *each* time we create a new build, all filenames will get altered and clients will have to re-download the whole application code again. So how can we guarantee that clients always get the latest versions of assets without re-downloading all of them?
+But the problem here is that, building after *any file update* will update all filenames and clients will have to re-download all application code again. So how can we guarantee that clients always get the latest versions of assets without re-downloading all of them?
 
 ## Generating unique hashes for each file
 
-What if we could produce the same filename if the contents of the file did not change between builds? For example, the file where we put all our libraries and other vendor stuff does not change that often.
+What if we could produce the same filename, if the contents of the file did not change between builds? For example, it would be unnecessary to re-download a vendor file, when no dependencies have been updated, only application code.
 
-T> Separate your vendor and application code with [CommonsChunkPlugin](/plugins/commons-chunk-plugin) and create an explicit vendor chunk to prevent it from changing too often.
+Webpack allows you to generate hashes depending on file contents, by replacing the placeholder `[hash]` with `[chunkhash]`. Here is the updated config:
```

Comment:
Also, I agree about the linting rule. `webpack` as first word in sentences will probably often be written with a capital 'W'.